torch/csrc/jit/codegen/cuda/arith.cpp

```diff
@@ -732,6 +746,54 @@ TensorView* clamp(TensorView* in, Val* min_val, Val* max_val) {
   return clamp(in->as<Val>(), min_val, max_val)->as<TensorView>();
 }
 
+// sum_to operator
+
+TensorView* sum_to(TensorView* in, const std::vector<Int*>& shape) {
```

Comment:
just a small nit, can we rename this to sum_to_size?

Reply:
Sure, just updated thanks.